Dockerfile

```
@@ -0,0 +1,7 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+RUN yum -y update; yum clean all
+
+RUN yum -y install boost-devel
```

Maybe better to call `yum clean all` after the install, as I believe some metadata will be generated during install.

Also maybe we can ensure that we don't execute next command if current one fails, and we just chain these 3 commands to have a single docker image layer for boost lib like:

```suggestion
RUN yum -y update && yum -y install boost-devel && yum clean all
```

---

https://github.com/Constructor-io/kenlm/pull/2/commits/01c8a69b7525fb337b6708012eb0a706e4e0e398